apps/codeforafrica/src/components/ErrorHero/ErrorHero.js

```diff
@@ -0,0 +1,65 @@
+import { RichTypography, Section } from "@commons-ui/core";
+import Grid from "@mui/material/Grid";
+import React from "react";
+
+import TwoToneBackground from "@/codeforafrica/components/TwoToneBackground";
+
+const ErrorHero = React.forwardRef(function ErrorHero(props, ref) {
+  const { title, subtitle, sx, tags, ...other } = props;
+
+  return (
+    <TwoToneBackground
+      sx={{ py: { xs: "43px", sm: 0, md: "65px", lg: 0 } }}
+      {...other}
+      ref={ref}
+    >
+      <Section
+        sx={{
+          maxWidth: {
+            sm: "648px",
+            md: "912px",
+          },
+          px: { xs: 2.5, sm: 0 },
+          py: { xs: 5, md: "25px" },
+          zIndex: 1,
+          ...sx,
+        }}
+      >
+        <Grid
+          container
+          flexDirection={{ xs: "column", md: "row" }}
+          alignItems={{ md: "center" }}
+          rowSpacing={2.5}
+          justifyContent={{ md: "space-between" }}
+        >
```

Comment:
Not sure `Grid` adds anything here. This component is very similar to [`PageHeader`](https://github.com/CodeForAfrica/ui/blob/main/apps/codeforafrica/src/components/PageHeader/PageHeader.js), just different `RichTypography` variants.